OrbitLinuxTracing/Utils.h

```diff
@@ -32,20 +34,26 @@ inline std::string ExecuteCommand(const std::string& cmd) {
   return result;
 }
 
+inline std::optional<std::string> ReadFile(std::string_view filename) {
```

Comment:
Why is this inline?

Reply:
This seems to be common for all functions in this file. Is there any reason why this is done? If not, I would suggest to move them to Utils.cpp. Some of them should probably be in an anonymous namespace, if they are not used outside of Utils.h/cpp.